ui/app/src/components/DeletePluginDialog/UninstallPluginDialogContainer.tsx

```diff
@@ -48,7 +48,10 @@ export function UninstallPluginDialogContainer(props: UninstallPluginDialogConta
         });
         onComplete?.();
       },
-      error: (response) => {
+      error: ({ response: { response } }) => {
+        callbacksRef.current.onSubmittingAndOrPendingChange({
+          isSubmitting: false
+        });
```

Comment:
What happens after an error? Can you retry? Is the UI in a coherent post-error state?

Reply:
When an error happens, an error dialog is displayed. After dismissing the dialog, you can retry uninstalling (now that submit state is reset).